src/main/menu.js

```diff
@@ -11,145 +11,148 @@ import { openFile, openFileForImporting, newFile } from './lib/files';
 import { getWindowManager } from './lib/window-manager';
 import { checkForUpdates } from './lib/updater';
 import { getMainWindow } from './utils/window';
+import i18n from '../shared/i18n';
 import pkg from '../../package.json';
 
-const defaultTemplate = [
```

Comment:
Is this diff block because you moved this constant to the setup function?

Reply:
Yes, because I need the `store` variable for all menu points.